GraphicsToolsUnityProject/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.36f1
-m_EditorVersionWithRevision: 2020.3.36f1 (71f96b79b9f0)
+m_EditorVersion: 2021.3.3f1
```

Comment:
Unfortunately, Graphics Tools requires 2020 support. :( Would you be able to downgrade these changes to [2020.3.36f1](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/main/GraphicsToolsUnityProject/ProjectSettings/ProjectVersion.txt)?